### DISCUSSION TITLE
Handling wandering monsters

### DISCUSSION THREAD

**Discussion by eegeeZA** [loathers/garbage-collector#1689] [Q&A] [answered]
How does Garbo handle wandering monsters? For example, today on the Feast of Boris it runs without issues. But while the Aware of Bees is active, it will fail if it does encounter a bee.

**Comment by gausie** [comment]
It generally has handling for wandering monsters, but without looking into the code that just means that bees are not yet actively supported. This would be a reasonably good first contribution, if you want to make one!

**Reply by eegeeZA** [reply]
Happy make that PR if you can nudge me in the right direction of which file would be a good starting point.

**Reply by gausie** [reply]
Actually it's a tiny bit complicated, I might handle it in libram's `ifHolidayWanderer`

**Reply by eegeeZA** [reply, marked as answer]
I see that #2136 shoud solve this.
